source/_partials/content/hsts.html

```diff
@@ -0,0 +1,45 @@
+Using the <a href="/docs/pantheon-yml/"><code>pantheon.yml</code></a> file, you can set the use and duration of the HSTS header, and its affect on subdomains. You can set a short (5 minute) duration header, or a long (366 day) duration, and choose whether to affect subdomains. Using the long duration setting will help you get an A+ SSL rating from <a href="https://www.ssllabs.com/ssltest/" class="external">SSL Labs</a>.
+
+Use of the HSTS header is defined by the <code>enforce_https</code> directive, and takes five possible values:
+
+<ul>
+<li>No redirect and no HSTS header (default):
+
+    <pre><code class="hljs less">enforce_https: off</code></pre></li>
+
+<li>HTTPS redirect with a short HSTS duration (5 minutes), which does not apply to subdomains:
+
+    <pre><code class="hljs less">enforce_https: transitional</code></pre></li>
+
+<li>HTTPS redirect with a short HSTS duration (5 minutes), which also applies to subdomains:
+
+    <pre><code class="hljs less">enforce_https: transitional+subdomains</code></pre></li>
+
+<li>HTTPS redirect with a long HSTS duration (366 days, for an A+ SSL Labs rating), which does not apply to subdomains:
+
+    <pre><code class="hljs less">scheme: full</code></pre></li>
+
+<li>HTTPS redirect with a long HSTS duration (366 days, for an A+ SSL Labs rating), which also applies to subdomains:
+
+    <pre><code class="hljs less">scheme: full+subdomains</code></pre></li>
+</ul>
+
+Pantheon will send the following header based on the setting of <code>enforce_https</code>:
```

Comment:
Suggest something like: 

Pantheon configures the `Strict-Transport-Security` header according to the configuration of `enforced_https`: 

- `off` (default): HSTS is disabled, `Strict-Transport-Security` header is not set by Pantheon, HTTP will not forcefully redirect to HTTPS
- `transitional`: HSTS is enabled, `Strict-Transport-Security: max-age=300` is set by Pantheon, HTTP will redirect forcefully redirect to HTTPS
...etc

Reply:
thanks @rachelwhitton !
I tried to work your suggestions in wherever I could in the new new PR #4998